app/src/features/apiClient/helpers/apiClientExecutor/apiClientExecutor.ts

```diff
@@ -50,24 +50,28 @@ export class ApiClientExecutor {
     this.entryDetails.request.queryParams = [];
     this.renderedVariables = {};
 
-    const { headers, queryParams } = processAuthForEntry(
+    const processedAuth = processAuth(
       this.entryDetails,
       { id: this.recordId, parentId: this.collectionId },
       this.apiRecords
     );
+    this.entryDetails.auth = processedAuth;
+
+    const { renderVariables } = this.internalFunctions;
+    const { renderedEntryDetails, renderedVariables } = renderVariables(this.entryDetails, this.collectionId);
+
+    this.entryDetails = renderedEntryDetails;
+    this.renderedVariables = renderedVariables;
```

Comment:
does L#64 override L#58? 

Regardless, for clarity, can we move L#58 after this